core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -273,24 +283,35 @@ public boolean testClassLoad(final String className, final String asTypeName)
   @Override
   public List<ActiveCompaction> getActiveCompactions(String tserver)
```

Comment:
Could drop the `t` in the variable name.

```suggestion
  public List<ActiveCompaction> getActiveCompactions(String server)
```